bazel/test/python_test_repo/BUILD

```diff
@@ -60,3 +60,54 @@ py_test(
     ],
     python_version = "PY3",
 )
+
+# Test compatibility of py_proto_library and py_grpc_library rules with
+# proto_library targets as deps when the latter use import_prefix and/or
+# strip_import_prefix arguments
+proto_library(
+    name = "helloworld_moved_proto",
+    srcs = ["helloworld.proto"],
+    deps = [
+        "@com_google_protobuf//:duration_proto",
+        "@com_google_protobuf//:timestamp_proto",
+    ],
+    import_prefix = "google/cloud",
+    strip_import_prefix = ""
```

Comment:
`strip_import_prefix` I think `strip_import_prefix = ""` is same as not specifying it at all, but I just put it here to make it more explicit (and if somebody searches for this prefix as an example they may find the relevant test case). To put something reasonable in `strip_import_prefix` we need to have some prefix (note that this happens on top-level of the `python_test_repo` "sub-workspace" so there is no any prefix to strip). `strip_import_prefix` and `import_prefix` are about same thing, so testing one one of them should be enough. Otherwise I would need to create subpackage here with new protos and stuff... it just felt like too much of unnecessary garbage in this PR).